system-test/system_test.sh

```diff
@@ -35,7 +35,7 @@ for i in ${NODE_VERSIONS[@]}; do
 
   # Test support for accurate line numbers with node versions supporting this
   # feature.
-  if [ "$i" != "8" ] && [ "$i" != "10" ] && [ "$i" != "11" ]; then
+  if [ "$i" != "10" ] && [ "$i" != "11" ]; then
```

Comment:
Does Node 12 support this?

Reply:
Node 12 and the V8 canary version support accurate line numbers, but node 8, 10, and 11 did not support this.